restapi/src/test/java/io/stargate/web/docsapi/DocsApiTestSchemaProvider.java

```diff
@@ -83,7 +84,7 @@ public DocsApiTestSchemaProvider(int maxDepth, String keyspaceName, String colle
           ImmutableColumn.builder()
               .name(QueryConstants.P_COLUMN_NAME.apply(i))
               .type(Column.Type.Text)
-              .kind(PartitionKey)
+              .kind(Clustering)
```

Comment:
:warning: Wait is this correct change? You are changing the schema here?

Reply:
No, I believe old code was a mistake here. `pNNN` columns are clustering keys, not partition keys.

The reason why I changed it now, is that it breakes the new validation logic in `QueryExecutor`.

Reply:
OK but once we redeploy this will only influence new tables right? What about the existing tables? If it brakes the validation logic will it not work in the existing tables?

Reply:
Sorry, I'm not sure I understand... the `key` columns has always been the only partition key, the `pNNN` have always need clustering keys. This change affects only the test code. Production code is not affected and should work as before.

Reply:
Fu*** I did not see this is a test class.. Sorry my bad :exploding_head: